src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -327,8 +327,8 @@ private AdsDataProvider() {}
   @DataProvider
   public static Object[][] disableGptOasis() {
     return new Object[][]{{WIKI_SPECIAL, "SyntheticTests/ProvidersChain",
-                           "InstantGlobals.wgAdDriverEvolve2Countries=[XX]", AdsContent.TOP_LB,
-                           "Evolve2"},};
+                           "", AdsContent.TOP_LB,
```

Comment:
I don't know how this work but it feels to me that it could also work without those empty strings.